package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "eslint-plugin-yola",
+  "version": "0.1.0",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint -c .eslintrc.js ./**/*.js",
+    "test": "jest",
+    "prepublish": "npm run lint && npm run test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/yola/eslint-plugin-yola.git"
+  },
+  "author": "Yola Engineering <engineers@yola.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/yola/eslint-plugin-yola/issues"
+  },
+  "homepage": "https://github.com/yola/eslint-plugin-yola#readme",
+  "dependencies": {
+    "babel-eslint": "8.x",
+    "eslint": "4.x",
+    "eslint-config-airbnb": "16.x",
+    "eslint-config-airbnb-base": "12.x",
+    "eslint-plugin-import": "2.x",
+    "eslint-plugin-jsx-a11y": "6.x",
+    "eslint-plugin-react": "7.x",
+    "lodash": "^4.17.4"
+  },
+  "devDependencies": {
+    "eslint": "4.x",
```

Comment:
From installer:
> Dependency 'eslint' exists in both dependencies and devDependencies, using 'eslint@4.x' from dependencies

Reply:
Uh oh how did I miss that. It should be dev and peer